test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -180,3 +180,13 @@ Describe "Validate that Get-Help returns provider-specific help" -Tags @('CI', '
         }
     }
 }
+
+Describe "Validate about_help.txt under culture specific folder works" -Tags @('CI') {
+
+    It "Get-Help about_should should return help text and not multiple HelpInfo objects" {
+
+        $help = Get-Help about_should
```

Comment:
`about_should` belongs to Pester so we wouldn't rename it.  I can create test only help files to replace this.  I agree it would be more clean that way.